docs/variant-specific/pink.mdx

```diff
@@ -292,7 +292,7 @@ These conventions apply to any variant with a pink (touched by all ranks) suit.
 ### The Bubblegum Bluff (Two-Away-From-Playable Form)
 
 - Normally, _Bubblegum Bluffs_ are always performed by touching _one-way-from-playable_ pink cards.
-- Rarely, it is possible to perform a _Bubblegum Bluff_ on a _two-away-from-playable_ pink card. However, this can only be done in a specific situation where if the clued card to be _one-way-from-playable_, it would not work out.
+- However, it is sometimes possible that the lowest-possible, useful value is actually _two-away-from-playable_.
```

Comment:
just a heads up that the language "lowest-possible, useful value" here might be incomplete/lack context if we aren't also taking the rewrite suggestion for the first line.

Reply:
"...lowest-possible, useful value of a focused pink card is actually..." ?